### PR TITLE
Feature/remove generated pojo

### PR DESCRIPTION
**Pull request by benz-ubique**
Replace generated POJO with Kotlin classes, to provide better readability

### CODE REVIEW

**Comment by goebelUB** [comment]
[This line](https://github.com/admin-ch/CovidCertificate-SDK-Android/blob/397c6fd8f888b6dc80414acf3c4d57310a1bdab5/sdk/build.gradle#L16) can be deleted as well,

File: sdk/src/main/java/ch/admin/bag/covidcertificate/eval/euhealthcert/TestEntry.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import java.util.*
+
+@JsonClass(generateAdapter = true)
+data class TestEntry(
+	@Json(name = "tg") val disease: String,
+	@Json(name = "tt") val type: String,
+	@Json(name = "nm") val naaTestName: String?,
+	@Json(name = "ma") val ratTestNameAndManufacturer: String?,
+	@Json(name = "sc") val timestampSample: Date,
+	@Json(name = "dr") val timestampResult: Date?,
```
Comment (goebelUB) [inline sdk/src/main/java/ch/admin/bag/covidcertificate/eval/euhealthcert/TestEntry.kt:15]: Why have these as `Date`s but not the dates in the VaccinationEntry and RecoveryEntry?

File: sdk/src/main/java/ch/admin/bag/covidcertificate/eval/chain/CborService.kt [hunk 2]
```diff
@@ -35,8 +42,7 @@ internal object CborService {
 
 			map[CwtHeaderKeys.HCERT.AsCBOR()]?.let { hcert ->
 				hcert[keyEuDgcV1]?.let {
-					val eudgc = CBORMapper().readValue(getContents(it), Eudgc::class.java)
```
[the remark this answers is not in the document]
Comment (benz-ubique) [reply sdk/src/main/java/ch/admin/bag/covidcertificate/eval/chain/CborService.kt:38]: removed

File: sdk/src/main/java/ch/admin/bag/covidcertificate/eval/euhealthcert/Eudgc.kt [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import java.io.Serializable
+
+@JsonClass(generateAdapter = true)
+data class Eudgc (
+	@Json(name = "ver") val version: String,
+	@Json(name = "nam") val person: PersonName,
+	@Json(name = "dob") val dateOfBirth: String,
+	@Json(name = "v") val vaccinations: List<VaccinationEntry>?,
+	@Json(name = "t") val tests: List<TestEntry>?,
+	@Json(name = "r") val pastInfections: List<RecoveryEntry>?,
```
[the remark this answers is not in the document]
Comment (benz-ubique) [reply sdk/src/main/java/ch/admin/bag/covidcertificate/eval/euhealthcert/Eudgc.kt:14]: I will sync this with iOS and change it in a later commit

File: sdk/src/main/java/ch/admin/bag/covidcertificate/eval/euhealthcert/TestEntry.kt (hunk 1 above)
Comment (benz-ubique) [reply sdk/src/main/java/ch/admin/bag/covidcertificate/eval/euhealthcert/TestEntry.kt:15]: Was already like this in the generated Java classes. I will keep it like this for the moment

**Review by goebelUB — APPROVED** [review]

**Pull request merged**